src/util/doGet.ts

```diff
@@ -0,0 +1,26 @@
+import fetch from 'node-fetch'
+
+function isMasterAPIKey(apiKey:string) {
+  return apiKey.includes('.')
+}
+
+export default function (api:string, apiKey:string) {
+  return (url: string) => {
+    return fetch(`${api}/api/v1/${url}`, {
```

Comment:
Let's please resolve URLs using URL methods and not concatenating strings, to avoid problems with trailing slashes.